prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -361,6 +364,86 @@ ssize_t rxr_pkt_post_ctrl_or_queue(struct rxr_ep *ep, int entry_type, void *x_en
 	return err;
 }
 
+/*
+ * This function is used for any extra feature that does not have an alternative,
+ * for example FI_DELIVERY_COMPLETE
+ *
+ * This function will send a eager rtw packet with no data to trigger handshake
+ * to ensure the sending order and receiving order of rtm packets and
+ * minimize the cost for triggering.
```

Comment:
This comment is quite confusing. I think you meant that "the reason we use eager rtw instead of eager rtm is because receiver might require ordering and an eager rtm will interrupt the reorder process"